SampleReadme.md

```diff
@@ -0,0 +1,243 @@
+            "applicationId": "",
+            "apiKeys": {},
+            "id": "3"
+        },
+        {
+            "type": "cosmosdb",
+            "id": "8",
+            "name": "",
+            "serviceName": "",
+            "tenantId": "",
+            "subscriptionId": "",
+            "resourceGroup": "",
+            "endpoint": "https://ent67.documents.azure.com:443/",
+            "key": "",
+            "database": "",
+            "collection": ""
+        },
+        {
+            "type": "generic",
+            "id": "5",
+            "name": "ContentModerator",
+            "url": "",
+            "configuration": {
+                "key": "",
+                "region": ""
+            }
+        },
+        {
+            "type": "generic",
+            "id": "364",
+            "name": "Authentication",
+            "url": "",
+            "configuration": {
+                "Azure Active Directory v2": ""
+            }
+        },
+        {
+            "type": "luis",
+            "name": "",
+            "appId": "",
+            "authoringKey": "",
+            "subscriptionKey": "",
+            "version": "0.1",
+            "region": "westus",
+            "id": "120"
+        },
+        {
+            "type": "qna",
+            "name": "",
+            "id": "85",
+            "kbId": "",
+            "subscriptionKey": "",
+            "endpointKey": "",
+            "hostname": "https://{APP NAME}.azurewebsites.net/qnamaker"
+        },
+        {
+            "type": "dispatch",
+            "serviceIds": [
+                "120",
+                "85"
+            ],
+            "name": "",
+            "appId": "",
+            "authoringKey": "",
+            "subscriptionKey": "",
+            "version": "Dispatch",
+            "region": "westus",
+            "id": "152"
+        }
+    ],
+    "padlock": "",
+    "version": "2.0"
+}
+```
+
+## Run this Sample
+
+### Visual Studio
+- Navigate to the samples folder (`botbuilder-samples/samples/csharp_dotnetcore/{SAMPLE NAME}`) and open {SAMPLE NAME}.csproj in Visual Studio .
+- Run the project (press `F5` key)
+
+### Visual Studio Code
+- Open `botbuilder-samples/samples/csharp_dotnetcore/{SAMPLE NAME}` sample folder.
+- Bring up a terminal, navigate to `botbuilder-samples/samples/csharp_dotnetcore/{SAMPLE NAME}` folder.
+- Type `dotnet run`.
+
+## Testing the bot using Bot Framework Emulator
+[Microsoft Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+- Install the Bot Framework emulator from [here](https://github.com/Microsoft/BotFramework-Emulator/releases)
+
+### Connect to bot using Bot Framework Emulator **V4**
+- Launch Bot Framework Emulator
+- From the *File* menu select *Open Bot Configuration*
+- Navigate to your `.bot` file
+
+## Deploy to Azure
+
+### Using CLI Tools
+```bash

```

Comment:
Add an intro, such as:

To clone all of the services a bot into a new azure resource group: